docs/output.md

```diff
@@ -1054,6 +1054,25 @@ The plots will show:
   - Summary of the VEP run to be visualised with a web browser
   </details>
 
+#### NGSCheckMate
```

Comment:
Can you add this section to ToC as well? The way it is displayed now it follows the annotation section, but in the workflow it is more treated like a qc tool, so I would arguie it should go into the reporting section after mosdepth